content/kb/troubleshooting-volume-readonly-or-io-error.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "Troubleshooting: `volume readonly or I/O error`"
+author: Shuo Wu
+draft: false
+date: 2021-1-06
```

Comment:
The date is not recognized. https://deploy-preview-250--longhornio.netlify.app/kb/

Reply:
Fixed. Will Catherine review this PR later?